cruise-control/src/yaml/endpoints/load.yaml

```diff
@@ -0,0 +1,64 @@
+LoadEndpoint:
+  get:
+    operationId: load
+    summary: Get the cluster load once Cruise Control Load Monitor is running.
+    parameters:
+      - name: start
+        in: query
+        description: Start time of the cluster load.
+        schema:
+          type: long
+          format: int64
+          # Default is time of earliest valid window.
+      - name: end
+        in: query
+        description: End time of the cluster load.
+        schema:
+          type: long
+          format: int64
+          # Default is current system time.
+      - name: time
+        in: query
+        description: End time of the clutser load.
+        schema:
+          type: long
+          format: int64
```

Comment:
can we add a `minimum` and `default` field for this parameter?

Reply:
I added `minimum` for this parameter as per the comment. However, `default` for this field is the current time stamp at time of execution (which is difficult to express as a constant value). I omitted adding the field in the yaml.

Let me know if you have a strong preference for `default` here and I will investigate a way to add it in.